frontend/src/pages/my-projects/my-projects.tsx

```diff
@@ -70,25 +85,21 @@ const MyProjects: React.FC = () => {
     [dispatch, handleModalClose],
   );
 
-  const handleSearching = useCallback((value: string) => {
-    return setIsSearching(value.length > 0);
-  }, []);
-
   const handleSearch = useCallback(
     (data: ProjectGetAllParametersDto): void => {
-      void dispatch(projectActions.getUserProjects({ name: data.search }));
-      handleSearching(data.search);
+      setSearchName(data.search);
+      handleChangePage(PAGE_DEFAULT);
     },
-    [dispatch, handleSearching],
+    [handleChangePage],
   );
 
   const handleFormChange = initDebounce((event_: React.BaseSyntheticEvent) => {
     void handleSubmit(handleSearch)(event_);
   });
 
   const isProjectShow = useMemo(() => {
```

Comment:
should be part of the usePagination but with better name

Reply:
@what1s1ove  changed name. but didn't move it to usePagination, because it's not part of pagination logic. it's for showing all projects or "There are no projects" component